### ISSUE TITLE
Intuition behind validation accuracy being much lower than test accuracy

### ISSUE THREAD

**Issue by svobora** [bwconrad/vit-finetune#4]
---
